cmd/metricscollector/v1beta1/file-metricscollector/main.go

```diff
@@ -159,7 +159,7 @@ func watchMetricsFile(mFile string, stopRules stopRulesFlag, filters []string) {
 	// Check that metric file exists.
 	checkMetricFile(mFile)
 
-	// Get Main proccess.
+	// Get Main process.
 	_, mainProcPid, err := common.GetMainProcesses(mFile)
```

Comment:
Should we also rename this func?

Reply:
I think `GetMainProcesses` is not a misspelling.

Reply:
Do we get the main process or processes?

Reply:
This function returns array with all running processes with the main one.
So I think we should name it `GetMainProcesses`, and since we are getting only the main process, the comment is correct.

Reply:
SGTM.